### PR TITLE
The Messenger: Fix missing rules for Double Swing Saws

### PR DESCRIPTION
**Pull request by alwaysintreble**
## What is this fixing or adding?
Fixes a missing rule on the double swing saws seal.

## How was this tested?
Wasn't, but there's no risk here. Only affects portal rando.

### CODE REVIEW

**Review by ScipioWright — APPROVED** [review]
Fix matches how other rules look.

**Comment by Exempt-Medic** [comment]
I feel like updating one of both of your vertical unit tests probably makes sense here

File: worlds/messenger/rules.py [hunk 1]
```diff
@@ -430,6 +432,9 @@ def __init__(self, world: "MessengerWorld") -> None:
             {
                 "Autumn Hills Seal - Spike Ball Darts":
                     lambda state: self.has_vertical(state) and self.has_windmill(state) or self.is_aerobatic(state),
+                "Autumn Hills Seal - Double Swing Saws":
+                    lambda state: self.has_vertical(state) or self.has_windmill(state) or
```
Comment (Exempt-Medic) [inline worlds/messenger/rules.py:436]: Does `self.has_windmill(state)` do anything here? Isn't that incorporated into `can_destroy_projectiles`?

Comment (alwaysintreble) [reply worlds/messenger/rules.py:436]: ah yeah good catch

**Review by Exempt-Medic — APPROVED** [review]

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**